File: 3 Periodo/POO/PBL01/main.py
if __name__ == '__main__':
    a = int(input("Digite o valor do primeiro termo da PA: "))
    r = int(input("Digite a razão da PA: "))
    n = int(input("Digite o número de termos da PA: "))

    t = a + (n-1) * r
    s = (a + t) * n/2
    print(f"Último termo da PA: {t}")
    print(f"Soma de todos os termos da PA: {s}")

File: 3 Periodo/POO/PBL01/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.println("Digite o valor do primeiro termo da PA: ");
        float a = s.nextFloat();

        System.out.println("Digite o valor da razão da PA: ");
        float r = s.nextFloat();

        System.out.println("Digite o número de termos da PA: ");
        float n = s.nextFloat();

        s.close();

        float t = a + (n - 1) * r;
        float soma = (a + t) * n/2;

        System.out.println("Último termo da PA: " + t);
        System.out.println("Soma de todos os termos da PA: " + soma);
    }
}